src/TTrigger.cpp

```diff
@@ -160,7 +160,7 @@ bool TTrigger::setRegexCodeList(QStringList regexList, QList<int> propertyList)
     bool state = true;
 
     for (int i = 0; i < regexList.size(); i++) {
-        if (regexList[i].size() < 1) {
+        if (regexList[i].isEmpty() && propertyList.at(i) != REGEX_PROMPT) {
```

Comment:
Is `regexList` a `QList` which has the better read-only `QList::at(`...`)` instead of the read-write `QList::operator[`...`]`?

Reply:
Yeah, adjusted.